cmake/GetGRpc.cmake

```diff
@@ -12,6 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## gRPC is not always available.
```

Comment:
So was gRPC previously sometimes unavailable but now we should always be able to link it?

Reply:
gRPC was sometimes a pain to compile because of SSL but now that we have the genny toolchain (vcpkg) it's a nop.